### DISCUSSION TITLE
Contribution graph widget issue on HyperOS

### DISCUSSION THREAD

**Discussion by ViceVpc** [community/community#140053] [Mobile]
### Select Topic Area

Product Feedback, Bug

### Body

A new activity widget crashes the com.miui.home process after trying to select an account to add widget to the home screen. It is not possible to add a widget.

**Device**: Xiaomi 13 lite (ziyi_global)
**OS**: Xiaomi HyperOS 1.0.7.0 ULLMIXM
**Crash Log**: ![Crashlog.txt](https://github.com/user-attachments/files/17179080/Crashlog.txt)

**Comment by Olda-Hal** [comment]
The same thing also happens with:
Device: Redmi note 12S
HyperOS: 1.0.9.0 UHZEUXM

All other widgets work as expected but the GitHub one causes issues

**Comment by stevepopovich** [comment]
Hey y'all! Thanks for writing in and adding that crash log. I will look into fixing this and get back to you if we get it fixed.

**Reply by brenoepics** [reply]
This is a duplicate of #139993

**Reply by KananGurbanov** [reply]
Still not fixed. 17.01.2025

**Reply by stevepopovich** [reply]
We haven't had a ton of time to look at this, but we believe that HyperOS needs to add support for Glance widgets. Not sure if they have plans for that
